src/components/BibPage/BibDetail.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  Link as DSLink,
+  Heading,
+  List,
+} from "@nypl/design-system-react-components"
+import styles from "../../../styles/components/BibDetails.module.scss"
+import RCLink from "../RCLink/RCLink"
+import type {
+  BibDetail,
+  Url,
+  LinkedBibDetail,
+  SubjectHeadingDetail,
+} from "../../types/bibTypes"
+import { displayRtl, isItTheLastElement } from "../../utils/bibDetailUtils"
+import type { ReactNode } from "react"
+
+interface BibDetailsProps {
+  details: BibDetail[] | LinkedBibDetail[]
+  heading?: string
+}
+
+const BibDetails = ({ details, heading }: BibDetailsProps) => {
+  return (
+    details?.length > 0 && (
+      <List
+        title={heading && <Heading level="three">{heading}</Heading>}
+        noStyling
+        type="dl"
+        className={styles.bibDetails}
+        sx={{ borderBottom: "none" }}
+      >
+        {details.map(
+          (detail: BibDetail | LinkedBibDetail | SubjectHeadingDetail) => {
+            if (!detail) return
+            if (detail.label === "Subjects") {
+              return buildCompoundSubjectHeadingElement(
+                detail as SubjectHeadingDetail
+              )
+            } else if ("link" in detail) {
+              return buildLinkedElement(detail as LinkedBibDetail)
+            } else {
+              return buildPlainTextElement(detail as BibDetail)
+            }
+          }
+        )}
+      </List>
+    )
+  )
+}
+
+const buildDetailElement = (label: string, listChildren: ReactNode[]) => {
+  return (
+    <>
+      <dt>{label}</dt>
+      <dd>
+        <List noStyling data-testId={label} type="ol">
+          {listChildren}
+        </List>
+      </dd>
+    </>
+  )
+}
+
+const buildPlainTextElement = (field: BibDetail) => {
+  const values = field.value.map((val: string, i: number) => {
+    const stringDirection = displayRtl(val)
+    return (
+      <li dir={stringDirection} key={i}>
+        {val}
+      </li>
+    )
+  })
+  return buildDetailElement(field.label, values)
+}
+
+const buildCompoundSubjectHeadingElement = (field: SubjectHeadingDetail) => {
+  const subjectHeadingLinksPerSubject = field.value.map(
+    (subjectHeadingUrls: Url[]) => {
+      return buildSingleSubjectHeadingElement(subjectHeadingUrls)
+    }
+  )
+  const values = subjectHeadingLinksPerSubject.map((subject, i) => (
+    <li key={`subject-heading-${i}`} data-testid="subjectLinksPer">
+      {subject}
+    </li>
+  ))
+  return buildDetailElement(field.label, values)
+}
+
+const buildSingleSubjectHeadingElement = (subjectHeadingUrls: Url[]) => {
+  const urls = subjectHeadingUrls.reduce((linksPerSubject, url: Url, index) => {
+    const divider = (
+      // this span will render as > in between the divided subject heading links
+      <span data-testid="divider" key={`divider-${index}`}>
```

Comment:
Can this be achieved with CSS (pseudoselector with a content attribute?). Not sure if it can but curious.